src/v3/src/util/getLanguageDirection.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+const rtlLanguageCodes = new Set([
+  'ar', // Arabic
+  'arc', // Aramaic
+  'arz', // Egyptian Arabic
+  'ckb', // Kurdish (Sorani)
+  'dv', // Divehi
+  'fa', // Persian
+  'ha', // Hausa
+  'he', // Hebrew
+  'khw', // Khowar
+  'ks', // Kashmiri
+  'ps', // Pashto
+  'sd', // Sindhi
+  'ur', // Urdu
+  'uz', // Uzbeki Afghanistan
+  'yi', // Yiddish
+]);
```

Comment:
I derived this list from the below sources: 
https://meta.wikimedia.org/wiki/Template:List_of_language_names_ordered_by_code
and 
https://www.loc.gov/standards/iso639-2/php/code_list.php

Reply:
Just for my understanding, is this an exhaustive list of RTL languages or is it just the RTL languages we want to support?

Reply:
This is meant to be an exhaustive list of all RTL languages.